kafka-avro-schema-quickstart/src/main/docker/Dockerfile.jvm

```diff
@@ -0,0 +1,54 @@
+####
```

Comment:
I believe you can remove the docker files. The guide does not reference them.

Reply:
OTOH, they should be fine to use if a user would like to, right?

Reply:
I will remove it, I get the motivation now

Reply:
removed